Makefile.am

```diff
@@ -0,0 +1,182 @@
+## Process this file with automake to produce Makefile.in
+#
+# Author: B.Irby (NASA/GSFC), based on the previous CFITSIO Makefile.in
+#
+
+lib_LTLIBRARIES = libcfitsio.la
+
+libcfitsio_ladir = $(pkgincludedir)
+libcfitsio_la_HEADERS =	fitsio.h fitsio2.h longnam.h drvrsmem.h \
+	cfortran.h f77_wrap.h region.h
+
+if !NOFORTRAN
+F77_WRAPPERS = f77_wrap1.c f77_wrap2.c f77_wrap3.c f77_wrap4.c
+else
+F77_WRAPPERS = 
+endif
```

Comment:
Careful, this will cause the source tarball to be lacking these source files if you configure without Fortran present. Instead, I suggest the following idiom:

```
F77_WRAPPERS = f77_wrap1.c f77_wrap2.c f77_wrap3.c f77_wrap4.c

...
libcfitsio_la_SOURCES = ...
...

if !NOFORTRAN
libcfitsio_la_SOURCES += $(F77_WRAPPERS)
endif
 
...

EXTRA_DIST = ... $(F77_WRAPPERS)
```